src/SayMore/UI/ComponentEditors/SessionBasicEditor.cs

```diff
@@ -619,5 +626,123 @@ private void HandlePeopleEditClick(object sender, EventArgs e)
 				break;
 			}
 		}
+
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Method to save the new AccessProtocol node and update the existing AccessProtocol node when option get changed
+		/// </summary>
+		/// <param name="accessCode">Previous AccessCode</param>
+		/// <returns>New AccessCode</returns>
+		/// ------------------------------------------------------------------------------------
+		private string HandleAccessProtocols(string accessCode)
+		{
+			string currentAccessCode = accessCode;
+			if (Program.CurrentProject == null) return currentAccessCode;
```

Comment:
I see that SayMore already has quite a few places that don't follow this, but in most of our SIL projects, we generally put the return on the following line, rather than on the same line as the if. I think that generally makes and early return more obvious, resulting in code that is easier to read.